packages/tooltips/stories/2-Tooltip.stories.tsx

```diff
@@ -0,0 +1,180 @@
+import { Tooltip, Title, Paragraph } from '@zendeskgarden/react-tooltips';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Alert, Title as AlertTitle } from '@zendeskgarden/react-notifications';
+
+export default {
+  title: 'Components/Tooltips',
+  subcomponents: { Tooltip, Title, Paragraph }
+} as Meta;
+
+type TOOLTIP_SIZE = 'default' | 'small' | 'medium' | 'large' | 'extra-large';
+
+const retrieveTooltipContent = (size: TOOLTIP_SIZE, type: 'light' | 'dark') => {
+  const shortValue = 'Lorem ipsum dolor';
+  const mediumValue = 'Lorem ipsum dolor sit amet, consect.';
+  const longValue =
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt' +
+    ' ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ' +
+    'ullamco laboris nisi ut aliquip ex ea commodo consequat.';
+
+  if (size === 'default') {
+    if (type === 'light') {
+      return longValue;
+    }
+
+    return shortValue;
+  }
+
+  if (size === 'extra-large' || size === 'large') {
+    return longValue;
+  } else if (size === 'medium') {
+    return mediumValue;
+  }
+
+  return shortValue;
+};
+
+interface IStoryProps {
+  placement:
+    | 'auto'
+    | 'top'
+    | 'top-start'
+    | 'top-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'end'
+    | 'end-top'
+    | 'end-bottom'
+    | 'start'
+    | 'start-top'
+    | 'start-bottom';
+  size: TOOLTIP_SIZE;
+  hasArrow: boolean;
+  forceVisibility: boolean;
+  delayMS: number;
+}
+
+export const Default: Story<IStoryProps> = ({
+  placement,
+  size,
+  hasArrow,
+  forceVisibility,
+  delayMS
+}) => {
+  return (
+    <Grid>
+      <Row alignItems="center" style={{ minHeight: 500 }}>
+        <Col md={6} style={{ textAlign: 'center' }}>
+          <Tooltip
+            content={
+              <>
+                <Title>Dark Tooltip</Title>
+                <Paragraph>{retrieveTooltipContent(size, 'dark')}</Paragraph>
+              </>
+            }
+            hasArrow={hasArrow}
+            placement={placement}
+            size={size === 'default' ? undefined : size}
+            delayMS={delayMS}
+            isVisible={forceVisibility ? true : undefined}
+            zIndex={1}
+            popperModifiers={{ preventOverflow: { boundariesElement: 'viewport' } }}
+          >
+            <Button>Default tooltip</Button>
+          </Tooltip>
+        </Col>
+        <Col md={6} style={{ textAlign: 'center' }}>
+          <Tooltip
+            content={
+              <>
+                <Title>Light Tooltip</Title>
+                <Paragraph>{retrieveTooltipContent(size, 'light')}</Paragraph>
+              </>
+            }
+            hasArrow={hasArrow}
+            placement={placement}
+            size={size === 'default' ? undefined : size}
+            type="light"
+            isVisible={forceVisibility ? true : undefined}
```

Comment:
Would this line be equivalent to `isVisible={forceVisibility}`?

Reply:
Since this is a controlled value, `isVisible={false}` would not allow the Tooltips to be shown even when hovered. `undefined` lets the user interact with the mouse in its default state.